runtime/Go/antlr/utils.go

```diff
@@ -343,6 +341,33 @@ func EscapeWhitespace(s string, escapeSpaces bool) string {
 	return s
 }
 
+func TerminalNodeToStringArray(sa []TerminalNode) []string {
+	st := make([]string, len(sa))
+
+	for i, s := range sa {
+		st[i] = fmt.Sprintf("%v", s)
+	}
+
+	return st
+}
+
+func PrintArrayJavaStyle(sa []string) string {
```

Comment:
Seems strange to me to refer to Java in the Go runtime. This is only here to work around a test problem. Also, "Print" suggests I/O to me. Would something like StringWithCommas (as in `String()` with commas) work better?